From a8c8f1c3d26014b634a8f1771d20a861a7f2fd50 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Wed, 24 Sep 2014 13:01:40 +0800
Subject: [PATCH] ENGR00332937-2 ARM: dts: imx6qdl-sabresd: add no-1-8-v
 property for usdhc

The sabresd board does not have external 1.8v/3.3v signal voltage
switch support for usdhc.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index a656f68..fdfdc30 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -801,6 +801,7 @@
 	bus-width = <8>;
 	cd-gpios = <&gpio2 2 0>;
 	wp-gpios = <&gpio2 3 0>;
+	no-1-8-v;
 	status = "okay";
 };
 
@@ -810,6 +811,7 @@
 	bus-width = <8>;
 	cd-gpios = <&gpio2 0 0>;
 	wp-gpios = <&gpio2 1 0>;
+	no-1-8-v;
 	status = "okay";
 };
 
@@ -818,6 +820,7 @@
 	pinctrl-0 = <&pinctrl_usdhc4>;
 	bus-width = <8>;
 	non-removable;
+	no-1-8-v;
 	status = "okay";
 };
 
-- 
1.8.0

